components/automate-chef-io/data/docs/api-static/02-meta-description.yaml

```diff
@@ -26,13 +26,13 @@ info:
     {
       "id": "9d7ae605-5b6a-4850-a12f-b5cb7fa732f5",
       "description": "New Token",
-      "value": "bww8EEpr39_eYMnQ2zybtrP9uzk=",
+      "value": "bww8EEpr39_eYMnQ2zyBtrP9uzk=",
       "active": false,
       "created": "2019-12-03T00:15:10Z",
       "updated": "2019-12-03T00:15:10Z"
     }
 
-    export TOKEN2=bww8EEpr39_eYMnQ2zybtrP9uzk=
+    export TOKEN2=bww8EEpr39_eYMnQ2zyBtrP9uzk=
```

Comment:
The spellchecker appears to consider any string of characters in the same case to be a potential word, so it thought `zybtr` was a misspelled word. This value does not appear to be significant, but rather than make up a fake token value I just changed one letter to uppercase to fool the spellchecker.

Reply:
Is this a common format in this project? We could probably also replace this with a regex recognizing base64 values instead. WDYT?

Reply:
After mentioning a regex to recognize base64 values I went and looked if that was possible. It looks like it is, but none of the examples I saw included `_` like the above value does, so I'm not sure how these values are being generated. I'll use your solution instead.

Reply:
Sounds good. Though, I recognize mine is a bit of kludge. 🤷‍♂

Reply:
Apparently the `_` thing is because of golang's implementation of base64:

```
The most common encoding is the "base64" encoding defined in RFC 4648 and used in MIME (RFC 2045) and PEM (RFC 1421). RFC 4648 also defines an alternate encoding, which is the standard encoding with - and _ substituted for + and /. 
```

https://golang.org/pkg/encoding/base64/

Supporting that would not be too difficult, but in a recent commit I made a change that caused more files to be spellchecked and those had failures from fragments of base64 encoded strings, which seems much more of a maintenance burden to support.